redis/src/lib.rs

```diff
@@ -415,6 +415,7 @@ mod pipeline;
 #[cfg_attr(docsrs, doc(cfg(feature = "acl")))]
 pub mod acl;
 
+/// module for async support.
```

Comment:
Can we put the existing doc comment at the top of aio/mod.rs? (Though maybe the line "experimental" doesn't need to be in there anymore)

Reply:
done

Reply:
I think we need to remove this comment though, else it will break the module description in the top-level docs.

Reply:
done